### PR TITLE
[[ Bug 20039 ]] Tutorial 'do it for me' should center new controls

### PR DESCRIPTION
**Pull request by livecodeali**
Tweak revIDECreateObject to default to center of target stack, and
use default positioning with tutorial 'do it for me' functionality.

### CODE REVIEW

File: notes/bugfix-20039.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# Position tutorial controlss in better location when using 'Do It For Me'
```
Comment (livecodepanos) [inline notes/bugfix-20039.md]: There is a typo  (`controlss`)

Comment (livecodeali) [reply notes/bugfix-20039.md]: Thanks!

**Comment by livecodepanos** [comment]
@livecode-vulcan review ok c1a5963

**Comment by livecode-vulcan** [comment]
💙 review by @livecodepanos ok c1a596343cdd2041eabd3cf6c7cb09a36a023063

**Pull request merged**